pkg/kubelet/images/types.go

```diff
@@ -39,6 +39,9 @@ var (
 
 	// ErrInvalidImageName - Unable to parse the image name.
 	ErrInvalidImageName = errors.New("InvalidImageName")
+
+	// ErrInvalidPullPolicy - Unknown/unsupported pullPolicy.
+	ErrInvalidPullPolicy = errors.New("InvalidPullPolicy")
```

Comment:
This also means that you don't need it, as well as the constants in the event.

Reply:
Removed from events and types!